packages/ui-chakra/src/base/Tab/StepTab.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Tab, Tabs, TabList, TabPanels } from "@chakra-ui/react";
+const stepPannelTitles = ["Step 1", "Step 2", "Step 3"];
+
+type TabProps = {
+  formStep: number;
+  setFormStep: React.Dispatch<React.SetStateAction<number>>;
+};
+export const StepTab = function FormTabControl({ formStep, setFormStep }: TabProps) {
```

Comment:
This is kool, but why do we need to store this function `FormTabControl` in variable `StepTab`?

Reply:
resolve by only using arrow function